examples/print_affinity.cpp

```diff
@@ -49,16 +49,16 @@ int main(int argc, char** argv)
 
     std::ostringstream thread_list;
 #ifdef _OPENMP
-#pragma omp parallel for ordered
```

Comment:
I'm not sure exactly what astyle is doing here, maybe it is indenting due to the appearance of the `for` keyword?  Probably better to just edit out open mp pragma changes